document/core/exec/runtime.rst

```diff
@@ -523,37 +642,37 @@ This definition allows to index active labels surrounding a :ref:`branch <syntax
    The selected label is identified through the :ref:`label index <syntax-labelidx>` :math:`l`, which corresponds to the number of surrounding |LABEL| instructions that must be hopped over -- which is exactly the count encoded in the index of a block context.
 
 
-.. index:: ! configuration, ! thread, store, frame, instruction, module instruction
+.. index:: ! configuration, ! thread, ! trace, store, event, frame, instruction, module instruction
 .. _syntax-thread:
+.. _syntax-trace:
 .. _syntax-config:
 
 Configurations
 ..............
 
-A *configuration* consists of the current :ref:`store <syntax-store>` and an executing *thread*.
+A *configuration* consists of the current :ref:`store <syntax-store>`, a *trace* of the store :ref:`events <syntax-evt>` that have occurred so far, and a set of executing *threads*.
```

Comment:
Depending on how we choose to define valid executions, we may end up deciding not to keep track of the events so far at the operational level. It's probably fine for now though.

Reply:
Yes, would have to think much harder how to word this otherwise.